Refresh.GameServer/Endpoints/ApiV3/DataTypes/Response/ApiGameSkillRewardResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Refresh.GameServer.Types.Levels.SkillRewards;
+
+namespace Refresh.GameServer.Endpoints.ApiV3.DataTypes.Response;
+
+public class ApiGameSkillRewardResponse
```

Comment:
You should use the `IApiResponse` and `IDataConvertableFrom<>` interfaces, see ApiGameLevelResponse for an example